config.json

```diff
@@ -1308,32 +1308,32 @@
       {
         "groupId": "com.google.mlkit",
         "artifactId": "smart-reply",
-        "version": "17.0.0",
-        "nugetVersion": "117.0.0.2",
+        "version": "17.0.1",
+        "nugetVersion": "117.0.1",
         "nugetId": "Xamarin.Google.MLKit.SmartReply",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.mlkit",
         "artifactId": "translate",
         "version": "17.0.0",
-        "nugetVersion": "117.0.0.2",
+        "nugetVersion": "117.0.0",
         "nugetId": "Xamarin.Google.MLKit.Translate",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.mlkit",
         "artifactId": "vision-common",
         "version": "17.1.0",
-        "nugetVersion": "117.1.0.2",
+        "nugetVersion": "117.1.0",
```

Comment:
Same.

Reply:
this usually happens when I try to tune-in updates (based on dependencies) and sometimes I need to revert.

Usually I check this, but this time I forgot.

fixed